CHANGES.md

```diff
@@ -1,6 +1,437 @@
 Release Changes List
 ====================
 
+o spellcheck
+o quotes check
+o URL docs check (1.26)
+o linebreak check
+o check compiler flags / man pages
+o check example codes
+o remove blank sections
+
+version 1.26.0
+==============
+
+Twenty-ninth public release of Chapel, March 31, 2022
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* updated the Chapel compiler to support LLVM versions 11, 12, and 13
+* added a prototype ability to resize arrays of elements without default values
+* made stabilizing improvements to methods on ranges, domains, and arrays
+* added new 'CopyAggregation', 'Socket', and 'Channel' package modules
+* made stabilizing improvements to several standard library module interfaces
+* improved the portability, robustness, and performance of 'ofi' communication
+* expanded the generality of Chapel's ability to generate code for GPUs
+* added a new script for displaying available runtime configurations
+* many other bug fixes and improvements to error messages and documentation
+
+Packaging / Configuration Changes
+---------------------------------
+* the Chapel compiler's LLVM back-end now supports versions 11, 12, and 13
+* when multiple supported LLVM versions are found, the latest one is now chosen
```

Comment:
This line could arguably be left out since this is the first release supporting multiple versions of LLVM (in the chplenv scripts at least, unless I am forgetting something?). I don't mind keeping it though; in a way it is as an elaboration on the previous bullet.